src/FSAL/FSAL_CORTXFS/handle.c

```diff
@@ -1319,10 +1364,10 @@ static fsal_status_t kvsfs_ftruncate(struct fsal_obj_handle *obj,
 				     struct stat *new_stat, int new_stat_flags);
 
 /* FSAL.setattrs2 */
-fsal_status_t kvsfs_setattrs(struct fsal_obj_handle *obj_hdl,
-			     bool bypass,
-			     struct state_t *state,
-			     struct attrlist *attrs)
+static fsal_status_t kvsfs_setattrs(struct fsal_obj_handle *obj_hdl,
```

Comment:
Shouldn't this be "static inline" ? @pratyush-seagate what you say ?

Reply:
Yes, better to add inline.

Reply:
But inline keyword is just a suggestion to compiler, which it can ignore.

Reply:
Agreed, but lets take up the advantage wherever compiler can.

Reply:
fixed.